Fix awkward sentence (#6135)
WeblateOrg/weblate: docs/faq.rst

on).

> If you've already ran into a merge conflict, the easiest way is to solve all

Reviewer: ```suggestion
If you've already ran into a merge conflict, the easiest way to solve all
```

> conflicts locally at your workstation : simply add Weblate as a remote

Reviewer 2: This reverts text before 9531c2a4920c7293bbf44acb0389f86f80d8ac0e (see https://github.com/WeblateOrg/weblate/pull/3178) by @comradekingu

Reviewer: That should have been an em dash. The colon is an error to.
Original sentence with a comma works.

Author: So : "the easiest way **is to** solve all conflicts locally on your machine, **is to** add Weblate as a remote"?
That sentence seems incorrect to me, but I'm not a native speaker

> conflicts locally at your workstation : simply add Weblate as a remote

Reviewer: ```suggestion
conflicts locally on your machine, is to add Weblate as a remote
```
